### PR TITLE
A feature to detect if taskbar is visible or hidden

### PR DESCRIPTION
**Pull request by nikitatg**
Greetings. There is simple implementation of feature for Windows to find out if system taskbar is visible or hidden when autohide option is turned on.
Tested on Windows 10 22H2 and Flutter 3.3.10 and 3.7.12

### CODE REVIEW

File: windows/windows_taskbar_plugin.cc [hunk 1]
```diff
@@ -43,6 +43,7 @@ class WindowsTaskbarPlugin : public flutter::Plugin {
   static constexpr auto kResetOverlayIcon = "ResetOverlayIcon";
   static constexpr auto kSetWindowTitle = "SetWindowTitle";
   static constexpr auto kResetWindowTitle = "ResetWindowTitle";
+  static constexpr auto kIsTaskbarVisible = "isTaskbarVisible";
```
Comment (alexmercerind) [inline windows/windows_taskbar_plugin.cc:46]: Please start with upper-case like other methods.

File: windows/windows_taskbar.cc [hunk 2]
```diff
@@ -199,3 +199,23 @@ bool WindowsTaskbar::ResetWindowTitle() {
   }
   return true;
 }
+
+bool WindowsTaskbar::IsTaskbarVisible() {
+  HWND hTaskbarWnd;
```
Comment (alexmercerind) [inline windows/windows_taskbar.cc:204]: Please follow [Google Style Guide for C++](https://google.github.io/styleguide/cppguide.html) (mainly naming of identifiers is concerned here, snake case for local identifiers).

Format the source files with clang-format:
https://github.com/alexmercerind/windows_taskbar/blob/b3273d58aab9d71f61b51a84d5c6fb6cf0821529/.clang-format#L1-L3

**Review by alexmercerind — CHANGES_REQUESTED** [review]
nits

**Comment by alexmercerind** [comment]
Thanks!

**Comment by nikitatg** [comment]
PR was updated. Thanks for hints!

**Pull request merged**